googleapi/googleapi.go

```diff
@@ -79,6 +79,8 @@ type Error struct {
 	Header http.Header
 
 	Errors []ErrorItem
+	// Err is a wrapped apierror.APIError, typically derived from this Error.
```

Comment:
I wonder if this should be private and we have a Wrap(error) method. Feels odd that we would say the error is a certain type but don't treat it as such. Thoughts?

Reply:
This reminds me that I wanted to discuss more generally how we publicize the wrapped `APIError` to users. Ideally we would show an example of the `errors.As` usage somewhere that they can copy/paste.

Reply:
Replaced `Error.Err` with `Error.Wrap()`